cli/wizard/commands/list-squid-token/index.ts

```diff
@@ -161,8 +161,8 @@ export type InterchainTokenDetails = {
   originalMinterAddress: string;
   axelarChainId: string;
   tokenId: string;
-  deploymentTxHash: string;
-  deployerAddress: string;
+  deploymentMessageId: string;
+  deployer: string;
   remoteTokens: RemoteInterchainToken[];
```

Comment:
```suggestion
  chains: RemoteInterchainToken[];
```

Reply:
this is just an interface for how the data shows up in the underlying API. if we want to change the underlying api, this will be a broader refactor we can follow up with later